lnwire/lnwire.go

```diff
@@ -684,3 +722,21 @@ func isValidPkScript(pkScript PkScript) bool {
 
 	return true
 }
+
+// extractCanonicalPadding is a utility function to extract the canonical
+// padding of a big-endian integer from the wire encoding (a 0-padded
+// big-endian integer) such that it passes btcec.canonicalPadding test.
+func extractCanonicalPadding (b []byte) []byte {
```

Comment:
Has this section been run through `gofmt`? From a glance it looks like it hasn't been because the indentation looks a bit shallow.